### PR TITLE
HDDS-9308. Dashboard for JVM metrics needs to be tracked across the cluster.

### PR DESCRIPTION
**Pull request by muskan1012**
## What changes were proposed in this pull request?

Created a new dashboard and added visualisations for all the required JVM Metrics

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-9308

## How was this patch tested?

The patch has been tested over a cluster. After ploting and configuring all mentioned JVM Metrics on Grafana, the dashboard looks like : 
image

### CODE REVIEW

**Comment by kerneltime** [comment]
cc @tanvipenumudy

**Review by kerneltime — APPROVED** [review]

**Review by tanvipenumudy — APPROVED** [review]
Thank you @muskan1012 for the contribution!

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - JVM Metrics.json [hunk 1]
```diff
@@ -0,0 +1,4703 @@
...
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 44,
+      "panels": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "igU0coMVz"
```
Comment (duongkame) [inline hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - JVM Metrics.json:44]: I guess to make it easier for people to import this, the datasource uid should not be included. 
There's a similar question posted on grafana https://github.com/grafana/grafana/issues/60769, exporting the dashboard with `Export for sharing externally` will help.
@muskan1012

**Review by duongkame — APPROVED** [review]
Thanks for the contribution @muskan1012. LGTM.

**Comment by muskan1012** [comment]
Thank you @kerneltime @duongkame @tanvipenumudy for reviewing it!

**Pull request merged**